assets/src/apps/authoring/components/Flowchart/toolbar/FlowchartHeaderNav.tsx

```diff
@@ -143,7 +150,8 @@ export const FlowchartHeaderNav: React.FC<HeaderNavProps> = () => {
 
   const questionType = getScreenQuestionType(currentActivity);
   const hasQuestion = questionType !== 'none';
-
+  const isLessonEndScreen =
```

Comment:
besides the enum idea, I notice there is also this: 
https://github.com/Simon-Initiative/oli-torus/blob/master/assets/src/apps/authoring/components/Flowchart/screens/screen-utils.ts#L58

```
const isEndScreen = (screen: IActivity): boolean =>
  screen.authoring?.flowchart?.screenType === 'end_screen';
```

so you could just potentially use the utils instead of doing the enum